pybossa/util.py

```diff
@@ -1010,22 +1010,28 @@ def get_user_pref_db_clause(user_pref, user_email=None):
     # expand user preferences as per sql format for jsonb datatype
     # single user preference with multiple value or
     # multiple user preferences with single/multiple values
-    _valid = ((k, v) for k, v in user_pref.items() if isinstance(v, list))
-    user_prefs = [{k: [item]} for k, pref_list in _valid
-                  for item in pref_list]
     assign_key = 'assign_user'
     location_key = 'locations'
     language_key = 'languages'
 
+    _valid = ((k, v) for k, v in user_pref.items() if isinstance(v, list))
+    turkey_spellings = {'Turkey', 'Türkiye', 'Turkiye'}
+    if any(turkey_spelling in user_pref[location_key] for turkey_spelling in turkey_spellings):
+        user_pref[location_key] = list(set(user_pref[location_key]).union(turkey_spellings))
+
+    user_prefs = [{k: [item]} for k, pref_list in _valid
+                  for item in pref_list]
+
```

Comment:
same as above. can this be avoided adding all supported version to config. With this, code would be centralized as well.

Reply:
We need to map the three spellings to each other. For example, if a user has their location preference set to 'Turkey' and a task has location preference 'Turkiye', there would not be a match. This code segment ensures that any Turkey spelling will match the others.